src/main/java/chatterbot/data/Deadline.java

```diff
@@ -0,0 +1,29 @@
+package chatterbot.data;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
Could this variable name be more descriptive? Maybe consider something like "deadline"